koku/api/report/queries.py

```diff
@@ -371,7 +386,7 @@ class ProviderMap(object):
                     'default_ordering': {'total': 'desc'},
                 },
                 'storage_by_project': {
-                    'aggregates': {
+                    'aggregate': {
```

Comment:
Per the plural singluar argument this now leaves us with aggegate, and annotations. One singular, one plural. 

The instance type mapping wasn't updated, so I suspect that will break.

Reply:
Fair point, but if we need to change it, all instances of it needs to change not just new instances. And, well, your storage work only created inconsistency by using a different string than was being used elsewhere in the map. So, I'm fixing the inconsistency that was introduced.

Let's circle back in a future PR to migrate all cases of `aggregate` to `aggregates`.